templates/components/settings/python/settings.py

```diff
@@ -75,26 +75,37 @@ def init_azure_openai():
     from llama_index.embeddings.azure_openai import AzureOpenAIEmbedding
     from llama_index.llms.azure_openai import AzureOpenAI
 
-    llm_deployment = os.getenv("AZURE_OPENAI_LLM_DEPLOYMENT")
-    embedding_deployment = os.getenv("AZURE_OPENAI_EMBEDDING_DEPLOYMENT")
-    max_tokens = os.getenv("LLM_MAX_TOKENS")
+    model = os.getenv("MODEL")
     api_key = os.getenv("AZURE_OPENAI_API_KEY")
-    llm_config = {
+    azure_endpoint = os.getenv("AZURE_OPENAI_ENDPOINT")
+    api_version = os.getenv("AZURE_OPENAI_API_VERSION") or os.getenv(
+        "OPENAI_API_VERSION"
+    )
+    deployment_name = os.getenv("AZURE_OPENAI_DEPLOYMENT")
+
+    azure_config = {
+        "model": model,
         "api_key": api_key,
-        "deployment_name": llm_deployment,
-        "model": os.getenv("MODEL"),
-        "temperature": float(os.getenv("LLM_TEMPERATURE", DEFAULT_TEMPERATURE)),
+        "azure_endpoint": azure_endpoint,
+        "api_version": api_version,
+        "deployment_name": deployment_name,
+    }
+
+    max_tokens = os.getenv("LLM_MAX_TOKENS")
+    temperature = os.getenv("LLM_TEMPERATURE", DEFAULT_TEMPERATURE)
+    llm_config = {
+        **azure_config,
         "max_tokens": int(max_tokens) if max_tokens is not None else None,
+        "temperature": float(temperature),
     }
     Settings.llm = AzureOpenAI(**llm_config)
 
     dimensions = os.getenv("EMBEDDING_DIM")
     embedding_config = {
-        "api_key": api_key,
-        "deployment_name": embedding_deployment,
-        "model": os.getenv("EMBEDDING_MODEL"),
+        **azure_config,
```

Comment:
ah just realize that the deployment of the embedding model is different than the one from the LLM we must have for TS and Python those variables:

```
 Field(   
        description="The Azure OpenAI LLM deployment to use",
        env="AZURE_OPENAI_LLM_DEPLOYMENT",
    )
  Field(
        description="The Azure OpenAI embedding deployment to use",
        env="AZURE_OPENAI_EMBEDDING_DEPLOYMENT",
    )
```